README.md

```diff
@@ -1565,6 +1565,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
    * [optimizely.com](https://www.optimizely.com) — A/B Testing solution, free starter plan, 1 website, 1 iOS and 1 Android app
    * [Microsoft PowerBI](https://powerbi.com) — Business Insights & Analytics by Microsoft. Free Plan offers limited use with 1 Million User licenses.
    * [quantcast.com](https://www.quantcast.com/products/measure-audience-insights/) — Unlimited free analytics
+   * [Row Zero](https://rowzero.io) - Blazingly fast, connected spreadsheet. Connect directly to data databases, S3, and APIs. Import, analyze, and graph millions of rows instantly.
```

Comment:
please state whats free (see other entries)

Reply:
Updated our commit with messaging about free offering. Apologies for missing that the first time.